src/mono/wasm/runtime/startup.ts

```diff
@@ -529,10 +516,16 @@ export async function mono_wasm_load_config(configFilePath: string): Promise<voi
             config = JSON.parse(json);
         }
         runtimeHelpers.config = config;
+        config.environment_variables = config.environment_variables || {};
+        config.assets = config.assets || [];
+        config.runtime_options = config.runtime_options || [];
+        config.globalization_mode = config.globalization_mode || GlobalizationMode.AUTO;
+
     } catch (exc) {
-        const errMessage = "failed to load config file " + configFilePath;
+        const errMessage = "Failed to load config file " + configFilePath;
         console.error(errMessage, exc);
```

Comment:
we're technically supposed to either use the 'failed to load config file %s' form here or not pass extra arguments to console.error